.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,50 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: checkboxes
+    attributes:
+      label: 🕵️ Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
+      options:
+        - label: I have searched the existing issues
+          required: true
+  - type: dropdown
+    attributes:
+      label: What version of spark are you using?
+      description: This will help us identify where & when the bug may have appeared
+      options:
+        - 1.0.0
+        - Other (please specify it)
```

Comment:
Is there a `required: true` available for this or do we want to allow users to report issue that are not tied to a version?

Reply:
Yep, Ì'll add it as a requirement